app/_src/gateway/kong-manager/auth/oidc/configure.md

```diff
@@ -75,6 +75,72 @@ The following are configuration parameters in `admin_gui_auth_conf` for `openid-
 | `ssl_verify`<br>*optional*     | Boolean   | false   | Verify identity provider server certificate.    |
 {% endif_version %}
 
+<!-- FIXME !!! Should DISPLAY the following content when >= 3.6.x -->
+
+{:.note}
+> **Note**: If you are using the configuration from prior versions, you might need to follow the [migration guide]
+> (/gateway/{{page.kong_version}}/kong-manager/auth/oidc/migrate) to review and update your configuration.
```

Comment:
I'm not sure if this link is correct. I would appreciate it if @lena-larionova could help double-check.